test/s57-converter.test.ts

```
@@ -503,6 +527,102 @@ describe('buildTippecanoeCommand (argv stays small regardless of layer count)',
+    const db = new DatabaseSync(file);
+    db.exec('CREATE TABLE metadata (name TEXT, value TEXT)');
+    db.prepare('INSERT INTO metadata (name, value) VALUES (?, ?)').run('name', name);
+    db.close();
+    return file;
+  }
+
+  function readName(file: string): string | null {
+    const db = new DatabaseSync(file);
+    const row = db.prepare("SELECT value FROM metadata WHERE name = 'name'").get() as
+      | { value: string }
+      | undefined;
```

ESLint

[error] Replace `|·{·value:·string·}⏎·····` with `{·value:·string·}`

(prettier/prettier)